content/How to integrate a Typography System in React Native the right way - Agustina Chaer.md 

```diff
@@ -0,0 +1,35 @@
+# How to integrate a Typography System in React Native the right way
+
+![Main image](images/rn_typography_main.jpg)
+
+## The Problem
+
+If there is one thing that most react native apps have in common is displaying text. Some simple apps show very little, but most apps display a lot. That is why it’s vital to put the right system in place, so adding and refactoring texts in your app is as simple as possible.
+
+Now, you might be thinking: why am I reading an article on how to manage such a simple component? It’s just text, right? Yes, but more often than not, you need to apply styles and properties to that text to make it look the way you want. When you have to do it for all the texts in your app, it adds up. It, of course, begs the question: how can I reduce the amount of code repetition?
+
+Over the years, I have seen my fair share of attempts of solving this problem, and I can finally say that I have landed on one solution that left me quite pleased.
+
+## The Solution
+
+The solution is quite simple, but it’s based on following best UI practices. What do I mean? Having predefined styles for headings, paragraphs, and other texts that you might need. What is important is that all texts of your app fall into one of those definitions and that their amount doesn’t get out of hand. It is good to keep the number of definitions short, so it’s easier to remember what to put when coding and your app is consistent in styling, which gives you usability and accessibility points.
+
+Without any further ado, here is the code of the component and supplementary files:
+
+https://gist.github.com/aguscha333/7de34fd9219213f1e5ff48db208d7d6e
+
+Once you have all of that in place, you can simply import the component and use it as follows:
+
+https://gist.github.com/aguscha333/11911e87d42a510a35461a6d927e0a83
+
+It’s as simple as that, and you will usually only have to apply extra styles to a Text component instance if you need to add spacing and sometimes color or wrapping.
+
+Taking the naming convention and hierarchy of text components from the web makes it very easy to understand and remember. As you may have noticed, with the use of prop-types it is very easy to catch if you misspelled a type when setting it on an instance of the component.
+
+Although adding custom fonts to your React Native project is out of the scope of this article, you might find this other article of ours where we go over that subject useful: [How to customize fonts in React Native](https://www.rootstrap.com/blog/how-to-customize-fonts-in-react-native/)
```

Comment:
maybe change other article to just one in order to avoid repetition

```suggestion
Although adding custom fonts to your React Native project is out of the scope of this article, you might find this one of ours where we go over that subject useful: [How to customize fonts in React Native](https://www.rootstrap.com/blog/how-to-customize-fonts-in-react-native/)
```